### DISCUSSION TITLE
Some errors during the local test of istio go fuzzing

### DISCUSSION THREAD

**Discussion by OwenWangbattle** [istio/istio#48262] [Q&A]
As I run the command in the Readme in tests/fuzz, there are something wrong:
How could I modify the files in oss-fuzz/infra/base-images/base-builder, such as compile_go_fuzzer and apply it to the compilation? When I run the command line in https://github.com/istio/istio/tree/master/tests/fuzz/README.md :

`python infra/helper.py build_fuzzers istio ~/go/src/istio.io/istio` 

There are something wrong during the running time. In istio.io/istio/tests/fuzz/oss_fuzz_build.sh,  some commands are like:

`compile_go_fuzzer istio.io/istio/tests/fuzz func1 func2`  

I set my GOPATH=$HOME/go. In oss-fuzz/infra/base-images/base-builder/compile_go_fuzzer, there are some $GOPATH to directly find the position of go sources. However, as I run  compile_go_fuzzer istio.io/istio/tests/fuzz func1 func2, the output is like, for example:
`/usr/local/bin/compile_go_fuzzer: line 28: cd: /root/go/src/istio.io/istio/tests/fuzz: No such file or directory`
` istio.io/istio/tests/fuzz`
` istio.io/istio/tests/fuzz`
 `Running go-fuzz -tags gofuzz -func FuzzNewBootstrapServer -o fuzz_new_bootstrap_server.a` `istio.io/istio/tests/fuzz` 
So I'm comfused about this. How to modify the compile_go_fuzzer and apply the modification into the compilation?

**Comment by rootsongjc** [comment]
I got the following error while run `python infra/helper.py build_fuzzers istio  ~/Workspace/go/src/istio.io/istio`.

```
...
+ cd /src/istio
+ sed -i 's/\"testing\"/\"github.com\/AdamKorcz\/go-118-fuzz-build\/testing\"/g' /src/istio/pkg/fuzz/util.go
sed: couldn't open temporary file /src/istio/pkg/fuzz/sedpLpTiM: Permission denied
ERROR:__main__:Building fuzzers failed.
```
